git-repository/tests/discover/mod.rs

```diff
@@ -31,8 +35,10 @@ mod existing {
         let path = git_repository::path::discover::existing(&dir)?;
         assert_eq!(path.kind(), Kind::WorkingTree);
         assert_eq!(
-            path.into_repository_directory(),
-            dir,
+            path.into_repository_directory()
+                .canonicalize()
+                .expect("should be a directory"),
+            dir.canonicalize().expect("should be a directory"),
```

Comment:
Something like this:

```suggestion
            path.into_repository_directory().canonicalize()?,
            dir.canonicalize()?,
```

Reply:
Ah, yep, sure. Happy to make that change.
I'm on mobile now; will do it when I get a chance a bit later today.

And thanks for looking into the CI issue. 🙂